### DISCUSSION TITLE
how to set `CMAKE_DEBUG_POSTFIX` during `xrepo install`?

### DISCUSSION THREAD

**Discussion by BH2WFR** [xmake-io/xmake#5599] [Q&A] [answered]
e.g.
`xrepo install -v -y -f "vs_runtime='MDd'" -m debug -k shared -a x64 cpr`

**Comment by waruqi** [comment]
你要设置这个来做什么 ，目前你只能改 package 配置，才能加 cmake 配置。。

可以自己工程里面，或者自己的私有仓库里，去覆盖 cpr 配置，重写 on_install 去传入。

**Comment by star-hengxing** [comment, marked as answer]
1. 开个单独的 xmake 工程，只写 `add_requires` 就行。
2. 把需要改的包描述，从官方包仓库抄下来定制
3. 用 [add_requireconfs](https://xmake.io/#/manual/global_interfaces?id=add_requireconfs) 对包更多的定制
